### PR TITLE
Home: Bottom two components

### LINKED ISSUE #103: Homepage - Implement Reach out / Help section
image

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #103 
copy is tbd pending #131

image
image
image

### CODE REVIEW

File: src/_includes/footer.html [hunk 1]
```diff
@@ -1,10 +1,27 @@
 <footer>
   <nav>
     <ul class="links">
-      <li><a rel="noreferrer" href="#">Back to top</a></li>
-      <li><a rel="noreferrer" target="_blank" href="https://dot.ca.gov/privacy-policy">Privacy policy</a></li>
-      <li><a rel="noreferrer" target="_blank" href="https://www.ca.gov/use/">Conditions of use</a></li>
-      <li><a rel="noreferrer" target="_blank" href="https://registertovote.ca.gov/">Register to vote</a></li>
+      <li>
```
Comment (machikoyasuda) [inline src/_includes/footer.html:4]: Just a spacing change. Not sure how this happened!

**Comment by machikoyasuda** [comment]
Should come after #149

**Comment by machikoyasuda** [comment]
@angela-tran Rebased `staging` and ready for review

**Review by angela-tran — APPROVED** [review]

**Comment by machikoyasuda** [comment]
Woohoo! 

image

**Pull request merged**